src/swagger-specs/destinations.yaml

```diff
@@ -1443,51 +2702,680 @@ components:
+        updatedBy:
+          type: string
+          title: Updated User
+          description: The Adobe ID of the Identity Management System (IMS) user who last updated the base connection. In the case of service tokens, the value for `createdBy` is the same as `createdClient`.
+        createdClient:
+          type: string
+          title: Created client
+          description: The Adobe Identity Management System (IMS) client who created the base connection. This can be an application or a user. For example, the value `exc_app` represents base connections created in the Experience Platform UI. 
+        updatedClient:
+          type: string
+          title: Updated client
+          description: The Adobe Identity Management System (IMS) client who last updated the base connection. This can be an application or a user. For example, the value `exc_app` represents base connections created in the Experience Platform UI. 
+        imsOrgId:
+          type: string
+          title: IMS org ID
+          description: The organization ID to which the base connection belongs.
+        sandboxId:
+          type: string
+          title: Sandbox ID
+          description: The ID of the sandbox in which the base connection was created.
+        sandboxName:
+          type: string
+          title: Sandbox Name
+          description: The name of the sandbox in which the base connection was created.
+        name:
+          type: string
+          title: Connection name
+          description: The name of the base connection.
+        etag:
+          type: string
+          title: eTag
+          description: The updated, unique eTag returned by the HTTP response. The eTag value updates with every successful update of a base connection.
+        state:
+          type: string
+          title: Connection state
+          description: >
+            Represents the current state of the base connection. A base connection can be `enabled` or `disabled`.
+          enum: [enabled, disabled]  
+        connectionSpec:
+          type: object
+          title: Connection Spec.
+          description: Request model for connection spec information.
+          properties:
+            id: 
+              type: string
+              title: Id
+              description: Connection spec ID. Each destination connector has a unique connection spec ID, which must be used when establishing base, source, or target connections for that specific destination.
+            version: 
+              type: string
+              title: Version
+              description: Connection spec version. This value is `1.0` unless clearly specified otherwise.
+        auth:
+          oneOf:
+            - $ref: '#/components/schemas/AmazonS3Auth'
+            - $ref: '#/components/schemas/SFTPWithPasswordBaseConnectionAuthParameters'
+            - $ref: '#/components/schemas/SFTPWithSSHKeyBaseConnectionAuthParameters'
+
```

Comment:
For properties that use `oneOf`, they should probably have a description that breaks down what determines the fields that will appear under `auth`, so they have more context into which version of the response they should be looking at.